src/test/regress/expected/multi_deparse_function.out

```diff
@@ -295,7 +295,7 @@ CONTEXT:  PL/pgSQL function deparse_and_run_on_workers(text) line XX at RAISE
 SELECT deparse_and_run_on_workers($cmd$
 ALTER FUNCTION add(int, int) SET TIME ZONE INTERVAL '-08:00' HOUR TO MINUTE;
 $cmd$);
-INFO:  Propagating deparsed query: ALTER FUNCTION function_tests.add(integer, integer) SET TIME ZONE INTERVAL '@ 8 hours ago';
+INFO:  Propagating deparsed query: ALTER FUNCTION function_tests.add(integer,integer) SET TIME ZONE INTERVAL '@ 8 hours ago';
```

Comment:
could this be an issue in ruleutils_14.c (e.g.,  an additional whitespace?)

Reply:
ping on this, all the difference seems like a single while space. Could it be possible that we add that whitespace so that we can avoid the additional output?

All the diff is consisting of these:
```
< INFO:  Propagating deparsed query: ALTER FUNCTION function_tests.add(integer,integer) SET "citus.setting;'" = 'hello '' world';
---
> INFO:  Propagating deparsed query: ALTER FUNCTION function_tests.add(integer, integer) SET "citus.setting;'" = 'hello '' world';
322c322
```

Where, the second (or later) parameters of the function has an additional/missing whitespace

Reply:
Looking at the pg code, this doesn't seem expected and in my local I don't get the extra space. Removed the alternative file and tests are still passing